src/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
 //*************************************************************************************************************
```

Comment:
If you would like to, it is fine to change the line that is full of stars `//**************************` to just `//`. It is the prevalent style that we use.